apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/BrazeTools/Hooks/useGenerateLocalBraze.ts

```diff
@@ -0,0 +1,135 @@
+import { useDispatch, useSelector } from "react-redux";
+import {
+  setPortfolioCards,
+  setActionCards,
+  setNotificationsCards,
+} from "~/renderer/actions/dynamicContent";
+import {
+  portfolioContentCardSelector,
+  actionContentCardSelector,
+  notificationsContentCardSelector,
+} from "~/renderer/reducers/dynamicContent";
+import {
+  PortfolioContentCard,
+  ActionContentCard,
+  NotificationContentCard,
+  LocationContentCard,
+} from "~/types/dynamicContent";
+
+const generateNewPortfolioCard = (
+  title: string,
+  description: string,
+  image: string,
+  order?: number,
+): PortfolioContentCard => ({
+  id: String(Date.now()) + "testinDev",
+  title,
+  description,
+  location: LocationContentCard.Portfolio,
+  image,
+  created: new Date(),
+  onClickOnSlide: () => {},
+  order,
+});
+
+const generateNewActionCard = (
+  title: string,
+  description: string,
+  image: string,
+  mainCta: string,
+  link: string,
+  secondaryCta: string,
+  order?: number,
+): ActionContentCard => ({
+  id: String(Date.now()) + "testinDev",
+  title,
+  description,
+  location: LocationContentCard.Action,
+  image,
+  mainCta,
+  link,
+  secondaryCta,
+  created: new Date(),
+  order,
+});
+
+const generateNewNotificationCard = (
+  title: string,
+  description: string,
+  cta: string,
+  viewed: boolean,
+  url?: string,
+  path?: string,
+  order?: number,
+): NotificationContentCard => ({
+  id: String(Date.now()) + "testinDev",
```

Comment:
what do you think of adding a `isMock` boolean to the `ContentCard` type instead of adding the `testinDev` in the id to identify mock cards ?